Watchman.Discord/Areas/Administration/Controllers/MutedUsersController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Devscord.DiscordFramework.Framework.Architecture.Controllers;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services;
+using Devscord.DiscordFramework.Services.Factories;
+using Watchman.Discord.Areas.Administration.BotCommands;
+using Watchman.Discord.Areas.Protection.Services;
+using Watchman.DomainModel.Messages;
+
+namespace Watchman.Discord.Areas.Administration.Controllers
+{
+    public class MutedUsersController : IController
+    {
+        private readonly MutingHelper _mutingHelper;
+        private readonly UsersService _usersService;
+        private readonly DirectMessagesService _directMessagesService;
+
+        public MutedUsersController(DirectMessagesService directMessagesService, MutingHelper mutingHelper, UsersService usersService)
+        {
+            this._directMessagesService = directMessagesService;
+            this._mutingHelper = mutingHelper;
+            this._usersService = usersService;
+        }
+
+        [AdminCommand]
+        public async Task MutedUsers(MutedUsersCommand mutedUsersCommand, Contexts contexts)
+        {
+            var mutedUsersList = await _usersService.GetUsersAsync(contexts.Server).ToListAsync();
+            var (title, description, values) = this.GetMuteEmbedMessage(mutedUsersList, contexts.Server.Id);
+            await _directMessagesService.TrySendEmbedMessage(contexts.User.Id, title, description, values);
+        }
+        private (string title, string description, Dictionary<string, Dictionary<string, string>> values) GetMuteEmbedMessage(List<UserContext> mutedUsersList, ulong serverId)
+        {
+            var title = "Lista wyciszonych użytkowników";
+            var description = "Wyciszeni użytkownicy, powody oraz data wygaśnięcia";
+            var values = new Dictionary<string, Dictionary<string, string>>();
+            foreach (var mutedUser in mutedUsersList)
+            {
+                var muteEvent = this._mutingHelper.GetNotUnmutedUserMuteEvent(serverId, mutedUser.Id);
+                if (muteEvent != null)
```

Comment:
odwróć warunek
```csharp
if (muteEvent == null)
{
    continue;
}

Reply:
ok, a dlaczego tak?

Reply:
zmniejsza zagnieżdżenie kodu oraz zwiększa czytelność przez to, że mniejsza ilość kodu jest w warunku, jak i również widać, że krok pętli się kończy jeśli warunek nie jest spełniony (nie trzeba śledzić co się dzieje dalej)

Reply:
ok, dziękuję :)